Bottom: 4dd125a021e086b82e6d26117b6cb498fcc3b9e8
Top:    35addde75bd544888b32441bf03effff7b0fb979
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-09 19:11:25 +0000

Refresh of change-unmodifed-parameters-to

---

diff --git a/src/action.c b/src/action.c
index b8707d1..8d5931a 100644
--- a/src/action.c
+++ b/src/action.c
@@ -7012,8 +7012,7 @@ static int
 parse_layout_attribute_units (char *name, int def)
 {
   const char *as, *units = NULL;
-  int n = 0, v;
-  bool r;
+  int n = 0, v, r;
 
   as = AttributeGet (PCB, name);
   if (!as)
@@ -7654,7 +7653,7 @@ ActionImport (int argc, char **argv, int x, int y)
 	}
       else if (ys)
 	{
-	  bool r;
+	  int r;
 	  x = GetValue (xs, units, &r);
 	  y = GetValue (ys, units, &r);
 	}
